doc/architecture/database_schemata.md

```diff
@@ -145,6 +191,70 @@ plant_detail }|--|| family : ""
+| **map_id**        | 1                                      | id of the map this list is for                                                 |
+| **name**          | Smoothie Ingredients                   | name of the list                                                               |
+| **description**   | Ingredients for my strawberry smoothie | description of the list                                                        |
+| **image**         | NULL                                   | an image for further customizing the list                                      |
+| **is_recurring**  | true                                   | a flag representing whether the objectives repeat themselves or not            |
+| **end_date**      | 2023-04-15                             | an optional date at which point the owner wants the objectives to be fulfilled |
+| **accomplished**  | 0                                      | the number of times the list was fulfilled; only relevant for recurring lists  |
+
+## `Ingredients`
+
+Many-to-many table to store relations between plants and ingredient lists.
+
+| **_Column name_** | **_Example_** | **_Description_**                                    |
+| :---------------- | :------------ | :--------------------------------------------------- |
+| **list_id**       | 1             | id of the ingredient list                            |
+| **plant_id**      | 1             | id of the plant used as an ingredient                |
+| **needed**        | 1             | the amount of this plant that is needed for the list |
+| **gathered**      | 0             | the amount of this plant that was already gathered   |
+
+## `Events`
+
+| **_Column name_** | **_Example_**        | **_Description_**                                                  |
+| :---------------- | :------------------- | :----------------------------------------------------------------- |
+| **id**            | 1                    | event id                                                           |
+| **map_id**        | 1                    | id of the map the event is taking place on                         |
+| **system_event**  | true                 | a flag representing whether this event is system or user generated |
+| **name**          | Harvest strawberries | name of the event                                                  |
+| **description**   | NULL                 | description of the event details                                   |
+| **event_date**    | 2023-04-15           | the date the event is taking place on                              |
+
+## `Favourites`
+
+Many-to-many table to store map-specific favourites.
+
+| **_Column name_** | **_Example_** | **_Description_** |
+| :---------------- | :------------ | :---------------- |
+| **map_id**        | 1             | id of the map     |
+| **user_id**       | 1             | id of the user    |
+| **plant_id**      | 1             | id of the plant   |
```

Comment:
Do we need to put `user_id` in here?
Wouldn't this be stored in the map and can be used from there?
Or od we want to allow collaborators to have their own set of favorites for a map?

Reply:
That is indeed the question.
@markus2330 what is your opinion on this? Should a user set their own favourites for any given map or is there only one set per map?

Reply:
There is only one set of plants in one map. It does not depend on the user.

As general principle: the screen with the map should look as similar as possible between users (for collaboration).